package.json

```diff
@@ -15,7 +15,8 @@
         "format": "prettier --write .",
         "format-staged": "npx pretty-quick --staged",
         "clean": "ts-node ./script/clean.ts",
-        "compile": "tsc --build --verbose && npm run postcompile --workspaces --if-present",
+        "compile": "npm run generateClients && tsc --build --verbose && npm run postcompile --workspaces --if-present",
+        "generateClients": " ts-node server/aws-lsp-codewhisperer/script/generateServiceClient.ts",
```

Comment:
I don't think `generateClients` belongs in root of the repository, since it's specific to CodeWhisperer server implementation. Could you move it to codewhisperer-server package please?